### PR TITLE
M302

### PR DESCRIPTION
**Pull request by Waterfox**
updating for new Heron

### CODE REVIEW

File: missfont.log [hunk 1]
```diff
@@ -0,0 +1,56 @@
+mktextfm Helvetica
```
Comment (tonybaltovski) [inline missfont.log]: Remove temp file please.

**Comment by tonybaltovski** [comment]
Looks like this repo is missing a gitignore file, otherwise LGTM.

**Comment by Waterfox** [comment]
Merge Merge Merge! 
Our manual is now 3 years out of date.

**Comment by tonybaltovski** [comment]
@Waterfox remove the temp files!

**Comment by Waterfox** [comment]
All the aux files removed. Please merge

**Pull request merged**